.devops/nix/package.nix

```diff
@@ -167,6 +169,9 @@ effectiveStdenv.mkDerivation (
         # TODO: Replace with autoAddDriverRunpath
         # once https://github.com/NixOS/nixpkgs/pull/275241 has been merged
         cudaPackages.autoAddOpenGLRunpathHook
+      ]
+      ++ optionals buildStatic [
+        glibc.static
```

Comment:
Shouldn't `buildStatic` (`static` if renamed) take precedence?

Reply:
I think @SomeoneSerge is correct - or more specifically `enableStatic` going with @philiptaron suggesting

Reply:
The thing is that there are other static builds using other libc implementations -- musl being a great example -- that don't require adding glibc static edition. I agree that the test should include enableStatic rather than my proposal of `stdenv.hostPlatform.isStatic`

Reply:
I see what you mean. I agree that adding `glib.static` to the hosting flake would work but the problem I guess is then that anybody wanting a static build would have to know that they need to do so. It feels a bit more explicit to have it here too. I've implemented these changes in d12f2399ac993a611884896c58b41db1cc5b88d3 and fixed some small issues so the branch now passes all tests on my end.